pkg/capabilities/consensus/ocr3/aggregators/identical.go

```diff
@@ -14,10 +14,10 @@ import (
 )
 
 type identicalAggregator struct {
```

Comment:
_nit_ touched, would be nice to add docstring of what is an `identicalAggregator`

Reply:
@bolekk how would you say `identicalAggregator` is different? One difference that I see is that it can enforce observations length. But maybe we can roll these together?